src/extension/background-script/connectors/lndhub.ts

```diff
@@ -375,9 +390,11 @@ export default class LndHub implements Connector {
         throw new Error(data.message);
       }
     }
+
     if (defaultValues) {
-      data = Object.assign(Object.assign({}, defaultValues), data);
```

Comment:
those default values are left-overs from joule, there default values had been used. we don't really do that I think.

Reply:
Removed it all connectors where it's not used. [Only connector were it's used is lnd](https://github.com/getAlby/lightning-browser-extension/blob/fix/960_expired-lndhub-alby-token/src/extension/background-script/connectors/lnd.ts/#L199-L205).